### ISSUE TITLE
A problem occurred evaluating project ':app'. Failed to apply plugin 'kotlin-android'. Kover requires extension with name 'androidComponents' for project ':app' since it is recognized as Kotlin+Android project

### ISSUE THREAD

**Issue by violantealessandro** [Kotlin/kotlinx-kover#520]
Hi, 

As described in the title, I'm trying to configure the latest Kover plugin version (0.7.5) on my project. 

The Android Gradle Plugin Version of my project is 6.9.

In the project, besides the Kover plugin, I use the following plugins:

`apply plugin: 'com.android.application'`
`apply plugin: 'kotlin-android'`
`apply plugin: 'kotlin-android-extensions'`
`apply plugin: 'kotlin-kapt'`
`apply plugin: 'realm-android'`

The error returned is the following:

> Failed to apply plugin 'kotlin-android'.
   > Kover requires extension with name 'androidComponents' for project ':app' since it is recognized as Kotlin+Android project
     Please create a bug in Kover Gradle Plugin bugtracker https://github.com/Kotlin/kotlinx-kover/issues

**Environment**
 - Kover Gradle Plugin version: 0.7.5
 - Gradle version: 6.9
 - Kotlin project type: Kotlin/Android
 - Coverage Toolset: Kover

**Comment by shanshin** [comment]
Hi, 
could you clarify the Android Gradle Plugin version? 
And also how exactly do you apply the Kover plugin?

**Comment by violantealessandro** [comment]
Hi,

The Android Gradle Plugin Version is 6.9.

That's how I include the Kover plugin.

```
plugins {
    id("org.jetbrains.kotlinx.kover") version "0.7.5"
}

apply plugin: 'com.android.application'
apply plugin: 'kotlin-android'
apply plugin: 'kotlin-android-extensions'
apply plugin: 'kotlin-kapt'
apply plugin: 'realm-android'
```

I've also tried including the plugins in the following way, but the error returned is the same:

```
plugins {
    id("org.jetbrains.kotlinx.kover") version "0.7.5"
    id("com.android.application")
    id("kotlin-android")
    id("kotlin-android-extensions")
    id("kotlin-kapt")
    id("realm-android")
}
```

**Comment by shanshin** [comment]
```
plugins {
    id("org.jetbrains.kotlinx.kover") version "0.7.5"
    id("com.android.application")
    id("kotlin-android")
    id("kotlin-android-extensions")
    id("kotlin-kapt")
    id("realm-android")
}
```
Could you try to change the order of application of plugins, for example, like this
```
plugins {
    id("com.android.application")
    id("kotlin-android")
    id("kotlin-android-extensions")
    id("kotlin-kapt")
    id("org.jetbrains.kotlinx.kover") version "0.7.5"
    id("realm-android")
}
```

**Comment by violantealessandro** [comment]
Hi,

I've done it, but the error returned is the same.

```
plugins {
    id("com.android.application")
    id("kotlin-android")
    id("kotlin-android-extensions")
    id("kotlin-kapt")
    id("org.jetbrains.kotlinx.kover") version "0.7.5"
    id("realm-android")
}
```

**Comment by shanshin** [comment]
Thanks!

**Comment by shanshin** [comment]
Fixed in 0.8.0-Beta.
This is not a production ready release, the plugin DSL is experimental and can be changed after receiving feedback.
